Build error on Linux and windows

I noticed that recently `sherpa-onnx` was [refactored](https://github.com/k2-fsa/sherpa-onnx/commit/a25075101c6beb735fd69af0b019775ddeb9305a) to allow using the c library without compile the binaries.
It fixed issues on macOS, however on `Windows` I still get the following compile error:

```console
 CMake Error at cmake_install.cmake:187 (file):
   file INSTALL cannot find
   "C:/Users/User/Documents/code/sherpa-rs/target/debug/build/sherpa-rs-sys-27a8ed25bdc9475f/out/build/lib/Release/sherpa-onnx-portaudio_static.lib":
   No error.
```
Full log:
[log.txt](https://github.com/user-attachments/files/16121031/error.txt)


Build options:
[build.rs](https://github.com/thewh1teagle/sherpa-rs/blob/main/sys/build.rs)

When setting `SHERPA_ONNX_ENABLE_BINARY` to `ON` on Windows, the problem solve and it compile without errors.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -263,7 +263,9 @@ include(simple-sentencepiece)
 set(ONNXRUNTIME_DIR ${onnxruntime_SOURCE_DIR})
 message(STATUS "ONNXRUNTIME_DIR: ${ONNXRUNTIME_DIR}")
 
-if(SHERPA_ONNX_ENABLE_PORTAUDIO)
+if(SHERPA_ONNX_ENABLE_PORTAUDIO AND SHERPA_ONNX_ENABLE_BINARY)
+  # portaudio is used only in building demo binaries and the sherpa-onnx-core
+  # library does not depend on it.
   include(portaudio)
 endif()
 

